src/MaterialsDesigner.js

```diff
@@ -90,7 +92,14 @@ class MaterialsDesigner extends mix(React.Component).with(FullscreenComponentMix
                                     editable={true}
                                     material={this.props.material}
                                     isConventionalCellShown={this.props.isConventionalCellShown}
-                                    onUpdate={this.props.onUpdate}
+                                    boundaryConditions={this.props.material.boundaryConditions}
+                                    onUpdate={(material) => {
+                                        // convert made material to MD material and re-set metadata
+                                        const config = material.toJSON();
+                                        const newMaterial = new Material(config);
+                                        newMaterial.metadata = this.props.material.metadata || {};
+                                        this.props.onUpdate(newMaterial);
+                                    }}
```

Comment:
I suggest moving this to Material class - createFromMadeMaterial - function

Reply:
Done.

Reply:
I meant passing metadata as a second optional parameter to createFromMadeMaterial also, but it's up to you